daemon/assets.py

```diff
@@ -48,48 +39,52 @@ def get_res_file(data, find_closest_with_url: bool = False):
     if f['fileType'] == 'blend':
       orig = f
       if data['resolution'] == 'blend':
-        # orig file found, return.
-        return orig, 'blend'
+        return orig, 'blend' # orig file found, return.
 
     if f['fileType'] == data['resolution']:
-      # exact match found, return.
-      return f, data['resolution']
-    # find closest resolution if the exact match won't be found.
+      return f, data['resolution'] # exact match found, return
+
+    # find closest resolution if the exact match won't be found
```

Comment:
@vilemduha I think the function searches for the closest resolution by default... If no exact match is done, then it searches for the closest. The parameter was just unused.